README.md

```diff
@@ -1,9 +1,47 @@
-# Learn Travis [![Build Status](https://travis-ci.org/dwyl/learn-travis.svg?branch=master)](https://travis-ci.org/dwyl/learn-travis)  [![Dependencies](https://david-dm.org/dwyl/learn-travis.png?theme=shields.io)](https://david-dm.org/dwyl/learn-travis) [![devDependency Status](https://david-dm.org/dwyl/learn-travis/dev-status.svg)](https://david-dm.org/dwyl/learn-travis#info=devDependencies)
+# Learn Travis
 
+<div align="center">
+    <a href="https://travis-ci.org/">
+        <img src="./images/travis-mascot.png" width="175">
```

Comment:
The mascot image _could_ be sourced directly from Travis-CI rather than uploading it to GH.
But not an issue. 👍